### PR TITLE
add beian option at footer for Chinese website owner

### PR DESCRIPTION
**Pull request by ifmos**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes have been added (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [x] Docs have been added / updated (for bug fixes / features).

## PR Type
**What kind of change does this PR introduce?**  

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number(s): N/A

## What is the new behavior?
...

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

This PR is going to add a beian option to show beian icp information at footer for Chinese website owners. In China, every legal website is requried to apply a series of beian icp information for every websites owned from the Communications Administration. So when we Chinese users get NexT theme for hexo, we have to change theme config to show the beian icp information. It troubles so much! So I create this pull request to add the option into _config.yml with footer.swing to show. 
### Files modified:
 _config.yml                  | 5 +++++
 layout/_partials/footer.swig | 3 +++
 2 files changed, 8 insertions(+)

### Global code changes:
* ADD: `beian` option in `_config.yml` - `theme` - `footer`.

### How to use?
In Next `_config.yml`:
```yml
footer:
    beian:
    # Beian icp information for Chinese users. In China, every legal website should have a beian icp in website footer.
    enable: true

    icp:
```

### How it looks?
image

Live demo [If Mos](https://ifmos.github.io/).

### CODE REVIEW

**Comment by ivan-nginx** [comment]
Wait a while. No need to close it.

**Comment by ifmos** [comment]
@ivan-nginx Is that enough to merge? This is my first pull request on Github, if there is something else required or this feature is useless? Plz tell me

**Comment by ivan-nginx** [comment]
> for Chinese website owners

I can't test this service because i'm not Chinese. Please, await for CN devels.

File: layout/_partials/footer.swig [hunk 1]
```diff
@@ -1,6 +1,9 @@
 <div class="copyright">{#
 #}{% set current = date(Date.now(), "YYYY") %}{#
-#}&copy; {% if theme.footer.since and theme.footer.since != current %}{{ theme.footer.since }} – {% endif %}{#
+#}{% if theme.footer.beian.enable %}{#
+#}<span class="with-love">{{theme.footer.beian.icp}}</span>{#
```
Comment (ivan-nginx) [inline layout/_partials/footer.swig]: Also, there is no need to add `span class="with-love`. Please, add only service here.

Comment (Raincal) [inline layout/_partials/footer.swig]: It's better to add a link to 'http://www.miitbeian.gov.cn/'

**Comment by sli1989** [comment]
@ifmos can u modify this pr according to the review? seems no other problems.

**Comment by sli1989** [comment]
The new commit is made.

**Comment by sli1989** [comment]
Sorry, for the force git commit.
I will open a new pr.

**Comment by ifmos** [comment]
@sli1989 Sorry for replying so late, I was busy before. The new pr will be ok! Thanks for your reminding~

**Comment by stevenjoezhang** [comment]
Superseded by https://github.com/theme-next/hexo-theme-next/pull/438

**Comment by stevenjoezhang** [comment]
@all-contributors please add @ifmos for code, review

**Pull request closed without merging**